src/SWP/Bundle/CoreBundle/Filter/DataFilter.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2020 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2020 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Filter;
+
+use SWP\Bundle\CoreBundle\Filter\Exception\KeyNotExistsException;
+use SWP\Bundle\CoreBundle\Filter\Exception\NotEqualDataException;
+
+class DataFilter
+{
+    protected $currentData;
+
+    protected $initialData;
+
+    public function loadData(array $data): self
+    {
+        $this->initialData = $data;
+        $this->currentData = $data;
+
+        return $this;
+    }
+
+    public function reset(): self
```

Comment:
this function is not used anywhere

Reply:
it's public one - so it's part of the api. It can be used for multiple checks, as `contains` changes verified data.  Example:
```
filter.contains('subject')
  .containsItem('code', '001')
  .containsItem('code', '123')
.reset()
.contains('custom_metadata')
  .containsItem('a', 'b')
```

filter returns true (match) as long as all checks don't throw exception.